R/fill.R

```diff
@@ -0,0 +1,22 @@
+#' Fill in missing values with the previous or following value
+#'
+#' `vec_fill()` fills gaps of missing values with the previous or following
+#' non-missing value.
+#'
+#' @param x A vector
+#' @param direction Direction in which to fill missing values. Must be either
+#'   `"down"` or `"up"`.
+#' @param leading A logical specifying if leading missing values should be
+#'   back-filled with the first non-missing value found in the chosen
+#'   `direction`.
+#' @param max_gap A single positive integer specifying the maximum size of the
```

Comment:
I don't love "gap" here because it somehow feels the wrong way around — like that's the maximum size gap that would be created. Maybe `max_fill` or just `max_n`?